package-lock.json

```diff
@@ -1,1982 +1,8 @@
 {
   "name": "setattributes",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Why did this file change? Are you on an old version of npm?

Reply:
I’m on NPM 6.14.7. I committed the file _as is_ after running `npm install`. It can be reverted. I have no idea how it managed to remove so much lines in the lock file.

Reply:
I think we should revert any changes to this file (unless dependencies change).

Reply:
Done!